vm/vm.go

```diff
@@ -188,6 +192,11 @@ func (inst *Instance) Info() ([]byte, error) {
 }
 
 func (inst *Instance) PprofPort() int {
+	if inst.hostFuzzer {
+		// In the fuzzing on host mode, fuzzers are always on the same network.
+		// Don't set up pprof endpoints in this case.
+		return 0
```

Comment:
These types could also provide PprofPort to disable pprof. Not sure what is better.

Reply:
Yes, hard to say. I chose this way not to duplicate this decision across the code, but I have no strong preference either.